lib/kitchen/verifier/inspec.rb

```diff
@@ -125,10 +125,27 @@ def local_suite_files
         Pathname.new(base).exist? ? [base] : []
       end
 
+      # Returns true if the local_suite_files and config[:inspec_tests] are
+      # pointing to the same directory
+      # @return [true] if test suites are duplicates
+      # @api private
+      def check_for_duplicates(local_suite_files, inspec_tests)
+        if local_suite_files[0]
+          local = local_suite_files[0].split('/').last(3).join('/')
+          return true if inspec_tests.include?(local)
+        end
+        false
+      end
+
       # Returns an array of test profiles
       # @return [Array<String>] array of suite directories or remote urls
       # @api private
       def collect_tests
+        puts 'local suite files', local_suite_files
+        puts 'config inspec tests', config[:inspec_tests]
+        # if check for duplicates returns true, only send back the value for config[:inspec_tests]
+        return config[:inspec_tests] if check_for_duplicates(local_suite_files, config[:inspec_tests])
+
         # get local tests and get run list of profiles
         (local_suite_files + config[:inspec_tests]).compact
```

Comment:
Okay, maybe we do the following:
1.) we rewrite local suite files to return a complex object like 
```
{:path=>"/Users/vjeffrey/test-kitchen-inspec/test/recipes/master"}
```
2.) We also resolve the `inspec_tests` to absolute paths before we hand them over to inspec
3.) Extend your `check_for_duplicates` to only work with objects?